user_guides/connector/pom.xml

```diff
@@ -40,14 +41,14 @@
         <asciidoctorj.pdf.version>1.5.0-alpha.16</asciidoctorj.pdf.version>
         <jruby.version>9.2.6.0</jruby.version>
         <!-- status: DRAFT, BETA, etc., or blank for final -->
-        <status>DRAFT</status>
+        <status></status>
```

Comment:
I don't think that status has to be emptied. I think you should keep `DRAFT` here.

Reply:
@edbratt @m0mus I think some of the TCKs have the 'DRAFT' removed - jpa, jsonp etc. Do you know when else should this status change to blank.

Reply:
I believe that we are updating to the final release, right?  To do those updates I'm following the [Guide](https://github.com/eclipse-ee4j/jakartaee-tck/wiki/Guide-For-Updating-the-TCK-User-Documentation-(Jakarta-EE-9)#jakartaee-tckuser_guidesyour-apipomxml) and some pull requests like #380

Reply:
I presume the DRAFT can be removed like it was removed for other tcks. @m0mus @edbratt Please mention if otherwise.